websub-ballerina/build.gradle

```diff
@@ -271,109 +174,14 @@ task updateTomlFiles {
     }
 }
 
-task revertTomlFiles {
+task revertTomlFile {
     doLast {
         ballerinaConfigFile.text = originalConfig
         ballerinaDependencyFile.text = originalDependencies
         ballerinaCompilerPluginFile.text = originalCompilerPlugin
     }
 }
```

Comment:
Is this function needed?

Reply:
Yes it is, otherwise .toml files are not reverted back

Reply:
Ack